bool calibrate
---
bool is_calibrated